packages/client/src/components/EllipseCarousel/EllipseCarousel.jsx

```diff
@@ -0,0 +1,117 @@
+import React, { useEffect, useState } from 'react';
+import { Box, IconButton, Typography } from '@mui/material';
+import Slider from 'react-slick';
+import ArrowBackIosIcon from '@mui/icons-material/ArrowBackIos';
+import ArrowForwardIosIcon from '@mui/icons-material/ArrowForwardIos';
+import { apiURL } from '../../apiURL';
+import styled from '@emotion/styled';
+
+export const EllipseCarousel = () => {
+  const [popularMovies, setPopularMovies] = useState([]);
+  const slider = React.useRef(null);
+
+  useEffect(() => {
+    const getPopularMovies = async () => {
+      try {
+        const response = await fetch(`${apiURL()}/movies`);
+        const data = await response.json();
+        setPopularMovies(data.slice(1, 12));
+      } catch (error) {
+        return error;
+      }
+    };
+    getPopularMovies();
+  }, []);
+
+  return (
+    <Section>
```

Comment:
why do you have a `Section` & `Container`

Reply:
@mohamedaldahoul 
I have removed Section and Container now, and using Container in mui instead 
I set Container MaxWidth to false because it is easier for me to move the MostPoppularText on the left side 

Yes, I dont need Section, in previous code, I tried to set Section to height 95 viewport to separate each section but after feature component merged, I dont need it now :) 

**What is added and edited** 

- Move useEffect to another file so this component can be reusable if needed in the future

- Edited some properties that using px to rem 

- Removed Section and Container and using mui Container 

Let me know if I need to fix somehting else?